### PR TITLE
feat: add resources dashboard for envoy gateway

### LINKED ISSUE #3675: Add support for monitoring envoy gateway memory and cpu in gateway-addon-chart
*Description*:
>Describe the desired behavior, what scenario it enables and how it
would be used.

https://gateway.envoyproxy.io/latest/tasks/observability/grafana-integration/ adds a dashboard for Envoy Proxy, but not for Envoy Gateway

[optional *Relevant Links*:]
>Any extra documentation required to understand the issue.

### PR DESCRIPTION
**Pull request by shawnh2**
**What type of PR is this?**

**What this PR does / why we need it**:

image

**Which issue(s) this PR fixes**:

Fixes #3675

### CODE REVIEW

File: site/content/en/latest/tasks/observability/grafana-integration.md [hunk 1]
```diff
@@ -65,6 +65,12 @@ This dashboard example shows the overall stats exported by Envoy Gateway fleet.
 
 ![Envoy Gateway Global: Infrastructure Manager](/img/envoy-gateway-global-infra-manager.png)
 
+### [Envoy Gateway Resources](https://raw.githubusercontent.com/envoyproxy/gateway/main/charts/gateway-addons-helm/dashboards/envoy-gateway-resource.json)
```
Comment (shawnh2) [inline site/content/en/latest/tasks/observability/grafana-integration.md:68]: doc-lint is failing cause the dashboard was uploaded along with doc update

**Review by arkodg — APPROVED** [review]
thanks !

**Pull request merged**